x-pack/plugins/lens/public/heatmap_visualization/visualization.tsx

```diff
@@ -170,6 +183,7 @@ export const getHeatmapVisualization = ({
           accessors: state.valueAccessor ? [{ columnId: state.valueAccessor }] : [],
```

Comment:
As you can configure this now, I guess we should show the current palette in the dimension trigger like for pie/xy/table?
<img width="534" alt="Screenshot 2021-06-22 at 13 36 43" src="https://user-images.githubusercontent.com/1508364/122918239-3507f580-d35f-11eb-817b-7858d9979f4f.png">

Reply:
Oh right, well spotted 😅